### PR TITLE
fixes #7737 - no specific logout URL needed, will go directly back to login.

### PR DESCRIPTION
**Pull request by adelton**

### CODE REVIEW

File: app/services/sso/form_intercept.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def login_url
     end
 
     def logout_url
-      controller.main_app.logout_users_path
+      return nil
```
Comment (orrabin) [inline app/services/sso/form_intercept.rb]: if you are just returning nil why not keep this empty?
you can just delete `controller.main_app.logout_users_path` without adding a return statement

Comment (isratrade) [inline app/services/sso/form_intercept.rb]: @orrabin, yes, overwriting a method without calling anything should be the same as `return nil`

Comment (adelton) [inline app/services/sso/form_intercept.rb]: > if you are just returning nil why not keep this empty?

I prefer to be explicit.

Comment (ares) [inline app/services/sso/form_intercept.rb]: +1 for being explicit, also just nil without return works for me

Comment (dLobatog) [inline app/services/sso/form_intercept.rb]: [Redundant returns](https://github.com/bbatsov/ruby-style-guide#no-explicit-return) is something [we're trying to fix](https://github.com/theforeman/foreman/blob/develop/.rubocop_todo.yml#L482)

**Comment by dLobatog** [comment]
Thanks @adelton , it's a great catch. Could you fix the issue with the `return`? It's ready for merge after that in my opinion.

**Comment by adelton** [comment]
Changed and rebased -> 50310781fde6f0ae5d752c57d3b1112c8f56a2ac.

**Comment by dLobatog** [comment]
Merged as https://github.com/theforeman/foreman/commit/f432ee3f50e124a2e11773c86345ef67db8f6340 

Thank you @adelton !

**Pull request closed without merging**